src/pages/claim-details/claim-actions.tsx

```diff
@@ -0,0 +1,141 @@
+import { useState } from 'react';
+import Button from '../../components/button';
+import { abbrStr, Claim } from '../../chain-data';
+import styles from './claim-actions.module.scss';
+import { formatApi3 } from '../../utils';
+import { isAfter } from 'date-fns';
+
+interface Props {
+  claim: Claim;
+}
+
+export default function ClaimActions(props: Props) {
+  const { claim } = props;
+  const [status, setStatus] = useState<'idle' | 'submitting' | 'submitted' | 'failed'>('idle');
+  const isPastDeadline = claim.deadline ? isAfter(new Date(), claim.deadline) : false;
+  const disableActions = !claim.open || isPastDeadline || status === 'submitting' || status === 'submitted';
+
+  // TODO DAO-151 Implement
+  const handleAcceptCounter = () => {
+    setStatus('submitting');
+  };
+
+  // TODO DAO-151 Implement
+  const handleAppeal = () => {
+    setStatus('submitting');
+  };
+
+  // TODO DAO-151 Add additional info messages for the different statuses
+  switch (claim.status) {
+    case 'Submitted':
```

Comment:
Nit: Consider adding enclosing the case body inside block scope (`{}`)

Reply:
Yeah I agree block scope can be very handy in switch statements, but I'm not so sure one should create the scope when it's not necessary?

Reply:
It's definitely not required. Feel free to keep it as is if you prefer.